plumbing/api.go

```diff
@@ -7,12 +7,12 @@ import (
 
 	"github.com/ipfs/go-bitswap"
 	"github.com/ipfs/go-cid"
-	"github.com/ipfs/go-ipfs-exchange-interface"
+	exchange "github.com/ipfs/go-ipfs-exchange-interface"
 	ipld "github.com/ipfs/go-ipld-format"
 	logging "github.com/ipfs/go-log"
 	uio "github.com/ipfs/go-unixfs/io"
-	"github.com/libp2p/go-libp2p-metrics"
-	"github.com/libp2p/go-libp2p-peer"
+	metrics "github.com/libp2p/go-libp2p-metrics"
+	peer "github.com/libp2p/go-libp2p-peer"
```

Comment:
These two changes should be unnecessary. Golang uses the last part of the path following the hyphen by default.

Reply:
My vs code setup does this on it's own. It will come back again on saving, will have to check.

Reply:
I am not sure how this exactly works, but what I saw right now by trying out is that 

`"github.com/libp2p/xyz"` would be replace by the package name, if different. say `abc "github.com/libp2p/xyz"`

if the package name is xyz it would just stay `"github.com/libp2p/xyz"`

Reply:
The package names of `go-libp2p-metrics` and `go-libp2p-peer` are already `metrics` and `peer`, so that shouldn't be a problem.

Either way, this isn't a blocking problem, so feel free to just leave it in.